packages/pds/src/app-view/api/app/bsky/unspecced.ts

```diff
@@ -192,6 +192,69 @@ export default function (server: Server, ctx: AppContext) {
     },
   })
 
+  server.app.bsky.unspecced.putNotificationPushToken({
+    auth: ctx.accessVerifier,
+    handler: async ({ auth, params }) => {
+      const { token, platform, appId, endpoint } = params
+      const {
+        credentials: { did },
+      } = auth
+      const notificationServer = ctx.notifServer
+      try {
+        await notificationServer.registerDeviceForPushNotifications(
+          did,
+          platform,
+          token,
+          endpoint,
+          appId,
+        )
+      } catch (error) {
+        throw new Error('Failed to register device push notification token')
+      }
```

Comment:
Is this gonna swallow earlier errors? I'm honestly unclear on how our error logs are reported, I'll look into that 🤔

Reply:
I am unclear too. @dholms can give some guidance

Reply:
Will let `kysley` handle that

Reply:
yeah this swallows earlier errors both from our perspective & from the clients perspective

if we throw errors from `@atproto/xrpc-server` those will get reported back to the client, everything else will get translated to an internal server error & logged for our sake